pkg/test/resourcefixture/testdata/basic/dataform/v1alpha1/dataformrepository/update.yaml

```diff
@@ -19,4 +19,8 @@ metadata:
 spec:
   projectRef:
     external: ${projectId}
-  region: us-west2
\ No newline at end of file
+  region: us-west2
+  workspaceCompilationOverrides:
```

Comment:
I think this is an optional field, so we'd better put it under the full test suite. https://cloud.google.com/dataform/reference/rest/v1beta1/projects.locations.repositories#Repository

Reply:
Also, can we updat the `gitRemoteSetting` in the full test suite to reflect the change in the update method?

Reply:
> Also, can we updat the `gitRemoteSetting` in the full test suite to reflect the change in the update method?

Let me double check this again but I'd say let's not just yet. The `gitRemoteSettings` update will probably pass in the mock but fail in the postsubmit as it requires an actual git repo.

> put it under the full test suite.

let me ask for some clarification offline

Reply:
Split the test suites into base and full! Let's follow up on the `gitRemoteSettings` if that's ok. If my testing shows any defaults w no side effects, we can add those as part of the full test suite